simplesat/package.py

```diff
@@ -68,6 +71,10 @@ def version(self):
     def dependencies(self):
         return self._dependencies
 
+    def __repr__(self):
+        return "PackageMetadata('{0}-{1}', key={2!r})".format(
+            self._name, self._version, self._key)
```

Comment:
I don't think we should print the key in the repr, as it really just an implementation detail (what defines identity, essentially)